### PR TITLE
Update pt-BR.json

### PR DESCRIPTION
**Pull request by zecaloteiro**

### CODE REVIEW

File: skill/locales/pt-BR.json [hunk 1]
```diff
@@ -1,105 +1,124 @@
...
+            {
+               "name":"VolumeLevelIntent",
+               "slots":[
+                  {
+                     "name":"VOLUMELEVEL",
+                     "type":"AMAZON.NUMBER"
+                  }
+               ],
+               "samples":[
+                  "definir volume em {VOLUMELEVEL}",
```
Comment (thorpelawrence) [inline skill/locales/pt-BR.json:66]:
```suggestion
                  "definir volume em {-|VOLUMELEVEL}",
```

A few places like this

**Review by thorpelawrence — COMMENTED** [review]
Hi, thanks for the PR - it looks like a few samples were removed, is there a reason for that?

**Comment by zecaloteiro** [comment]
Hi Thorpe, it wasn't making sense in Portuguese. The "pause" samples was
conflicting with the default "stop" command.
The previous file had some slots that didn't exists.


Thank you. Sorry about not include comments in the pr

Em sáb., 5 de nov. de 2022 07:10, Lawrence Thorpe ***@***.***>
escreveu:

**Comment by thorpelawrence** [comment]
Thanks, makes sense. There's just a few places with placeholders that might need to be updated left

**Comment by zecaloteiro** [comment]
Your skill is not working in PT-BR. After list the available devices it
can't do anything. I think that is the reason the skill has low ratings
among brazilian users.
The person who create this pt-br file may had a wrong understanding of
this. He replaced the name of the devices for numbers, etc, insert multiple
words between curly brackets (trying to use like options but the skill
hopes slots between brackets). If you get his "intends" and paste it in the
skill console it won't build.

After my adjustments it will work.



;)





Em sáb., 5 de nov. de 2022 09:04, Lawrence Thorpe ***@***.***>
escreveu:

**Pull request merged**